src/client/components/MealList.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState, useEffect} from "react";
+import {fetchMeals} from "../api";
+
+
+const MealList = () => {
+    const [meals, setMeals] = useState([]);
+
+    useEffect(() => {
+        fetchMeals()
+      .then(data => {
+          setMeals(data)
+      })
+
+    }, [])
+
+    return (
+        <div>
+            <h1>Meal List</h1>
+            {meals.map(meal => (
```

Comment:
Would be important to consider a situation where the meals are no there - not accessible / not fetched yet / etc., as right now in that case you would get an error 🙈